languages/github-link-pt_PT.po

```diff
@@ -20,15 +20,15 @@ msgstr ""
 
 #: github-link.php:71
 msgid "Visit GitHub repository"
-msgstr "Visitar o repositório do GitHub"
+msgstr "Visite o repositório do GitHub"
```

Comment:
Yes, after looking at it again, it makes more sense this way, changed to a more informal style, as it should be according WordPress translation guidelines.

Sometimes this can happen, specially with so few strings, after add some more, a greater picture of the context and logic can get a bit clear, than coherence and uniformization must be applied.